modules/storages/config/locales/en.yml

```diff
@@ -213,7 +213,8 @@ en:
     member_connection_status:
       connected: "Connected"
       connected_no_permissions: "User role has no storages permissions"
-      not_connected: "Not connected. The user should login to the storage via the %{files_label} tab of a work package to connect."
+      not_connected: "Not connected. The user should login to the storage via the following %{link}."
+      link: "link"
```

Comment:
🟡 don't we have some generic `link_label`?

Reply:
I do not know. I'll have a look.

Reply:
I have not found one. It seems like we do not have it.

Reply:
Maybe we should insert it at the same level as `label_list` in `config/locales/en.yml`? WDYT? It is a way to generic term for being so deeply nested.